### ISSUE TITLE
compare metadata given for Derby files against FAIR expectations

### ISSUE THREAD

**Issue by egonw** [bridgedb/BridgeDb#33]
The current output looks like:

    DATASOURCEVERSION    85
    DATATYPE    GeneProduct
    SERIES    Homo sapiens genes and proteins
    DATASOURCENAME    Ensembl
    BUILDDATE    20161019
    SCHEMAVERSION    3

This may or may not comply with FAIR (quick guess is: no, it does not).

**Comment by ariutta** [comment]
Good point -- it would be a big improvement to make this [FAIR](http://www.nature.com/articles/sdata201618) compliant!

The current full result for [this query](http://webservice.bridgedb.org/Human/properties):

```
DATASOURCEVERSION	2.0
DATATYPE	GeneProduct
SERIES	Homo sapiens genes and proteins
DATASOURCENAME	Ensembl
BUILDDATE	20161018
SCHEMAVERSION	3
DATASOURCEVERSION	1.0.0
DATATYPE	Interaction
SERIES	standard-interaction
DATASOURCENAME	EBI-RHEA
BUILDDATE	20161018
SCHEMAVERSION	3
DATASOURCEVERSION	HMDB3.6-CHEBI144-WIKIDATA2016101120161012
DATATYPE	Metabolite
SERIES	standard_metabolite
DATASOURCENAME	HMDB-CHEBI-WIKIDATA
BUILDDATE	20161012
SCHEMAVERSION	3
```

The same result in CSV format:

```csv
DATASOURCEVERSION,DATATYPE,SERIES,DATASOURCENAME,BUILDDATE,SCHEMAVERSION,
2.0,GeneProduct,Homo sapiens genes and proteins,Ensembl,20161018,3
1.0.0,Interaction,standard-interaction,EBI-RHEA,20161018,3
HMDB3.6-CHEBI144-WIKIDATA2016101120161012,Metabolite,standard_metabolite,HMDB-CHEBI-WIKIDATA,20161012,3
```

I'm sure we could make it even better, but at least using CSV or TSV would be a good start.

**Comment by AlasdairGray** [comment]
It would be great to see this follow the hcls community profile.

Alasdair J G Gray
Fellow of the Higher Education Academy
Assistant Professor in Computer Science
Herriot-Watt University, Edinburgh

www.macs.hw.ac.uk/~ajg33<http://<br/>www.macs.hw.ac.uk/~ajg33>

________________________________

**Comment by egonw** [comment]
See also http://www.fairmetrics.org/about.html

**Comment by egonw** [comment]
Last year, in the Google Summer of Code, Ammas worked on a VoID representation. This code is available in `master`.
